src/openrct2/world/Footpath.cpp

```diff
@@ -1382,7 +1385,7 @@ static void loc_6A6C85(
         return;
 
     if (tile_element_get_type(tileElement) == TILE_ELEMENT_TYPE_ENTRANCE) {
-        if (!entrance_has_direction(tileElement, (direction - tile_element_get_direction(tileElement)) & 3)) {
```

Comment:
Which this change? It can easily go under zero, so it does need this mask.

Reply:
Because I changed `entrance_has_direction` to take the mask into account, rather than the caller having to do it. The reason for this was to make the lines shorter.